superdwarf.lua

```diff
@@ -61,9 +61,15 @@ function main(argCommand)
         return
     end
 
-    local unit = dfhack.gui.getSelectedUnit()
+    unit = nil
```

Comment:
This should still be `local`, otherwise it will end up in the script's global scope.
```suggestion
    local unit = nil
```